number_box: 1
distance_hand: 0.02
discrete_side: 10
box_input: [0.062, 0.110, 0.094]
filename_out: "/box_004010020/D_062110094/"